src/main/java/org/jabref/rest/resources/RootResource.java

```diff
@@ -0,0 +1,32 @@
+package org.jabref.rest.resources;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSerializer;
+import com.google.gson.reflect.TypeToken;
+import org.jabref.logic.jabmap.MindMapManager;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntrySerializer;
+import org.jabref.model.entry.field.Field;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+@Path("/root")
+public class RootResource {
+
+    @GET
+    @Path("entries")
+    @Produces(MediaType.APPLICATION_JSON)
+    public String getEntries() {
+        List<BibEntry> entries = MindMapManager.instance().getBibEntries();
```

Comment:
This code is a bit strange. I would go to the current opened library directly instead of routing the request throught he `MindMapManager`. (Since one does not need to be that specific here, the more general "object" is enough)